shell/platform/tizen/tizen_renderer_evas_gl.cc

```diff
@@ -0,0 +1,102 @@
+// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "tizen_renderer_evas_gl.h"
+
+#include "flutter/shell/platform/tizen/tizen_log.h"
+
+TizenRendererEvasGL::TizenRendererEvasGL(TizenRenderer::Delegate& delegate,
+                                         int32_t x, int32_t y, int32_t w,
+                                         int32_t h)
+    : TizenRenderer(delegate) {
+  InitializeRenderer(x, y, w, h);
+}
+
+TizenRendererEvasGL::~TizenRendererEvasGL() { DestoryRenderer(); }
+
+void* TizenRendererEvasGL::SetupEvasWindow(int32_t x, int32_t y, int32_t w,
+                                           int32_t h) {
+  if (w == 0 || h == 0) {
+    FT_LOGE("Failed to create because of the wrong size");
+    return nullptr;
+  }
+  elm_config_accel_preference_set("hw:opengl");
+
+  evas_window_ = elm_win_add(NULL, NULL, ELM_WIN_BASIC);
+  elm_win_alpha_set(evas_window_, EINA_FALSE);
+  elm_win_aux_hint_add(evas_window_, "wm.policy.win.user.geometry", "1");
```

Comment:
If we don't need to resize the window on wearable environments, I think we can remove it.